partcad/src/partcad/cache_shape.py

```
@@ -0,0 +1,130 @@
+            for key, value in items.items():
+                # TODO(clairbee): if we know this is a valid solid, then we should use faster BREP
+                # if key.startswith(["part", "assembly"]):
+                #     # serialization = SERIALIZATION_BREP
+                # else:
+                serialization = self.serialization
+                if serialization == SERIALIZATION_PICKLE:
+                    data = pickle.dumps(value)
+                elif serialization == SERIALIZATION_BREP:
+                    data = Shape.to_brep(value)
+                else:
+                    raise ValueError(f"Unknown serialization type: {serialization}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)